CHANGELOG.md

```diff
@@ -43,7 +45,15 @@ Remember to align the itemized text with the first line of an item within a list
     See [a description of the versions](https://github.com/google/jax/blob/main/jax/experimental/jax2tf/README.md#native-serialization-versions).
     This change could break clients that set a specific
     JAX serialization version lower than 9.
-
+  * Parts of the {func}`jax.numpy.clip` API are undergoing deprecation to comply
+    with the array API 2023 standard:
+      * The unsupported `out` argument is now removed.
+      * The `a, a_min, a_max` arguments are deprecated in favor of `x, min, max`
+      instead.
+      * The new `x` argument is a positional-only argument.
+      * Passing complex-valued inputs to `x, min, max` is now deprecated.
+      * Passing `min=None, max=None` will no longer result in a `ValueError`
+        and instead will return an array with the same elements as the input.
```

Comment:
This is far more detail than we usually use for a changelog entry, but let's merge and fix later.